### ISSUE TITLE
The enumeration name for the remoting protocol versions should be named correctly

### ISSUE THREAD

**Issue by VipulM-MSFT** [Microsoft/service-fabric-services-and-actors-dotnet#81]
Currently the name of the enumeration that selects the remoting version are "RemotingClient" and "RemotingListener". This is confusing as these types are not representing the actual client or listener, but they are version selector. They should be appropriately named.

**Comment by Marusyk** [comment]
@VipulM-MSFT 
What do you think about this rename

```
public enum RemotingClientVersion
{
	V1 = 0,
	V2 = 1,
}
```

and

```
public enum RemotingListenerVersion
{
	V1 = 0,
	Compat = 1,
	V2 = 2,
}
```

**Comment by Marusyk** [comment]
Hi @VipulM-MSFT, any news?

**Comment by suchiagicha** [comment]
@Marusyk 

Looks good to me. Would you be interested in making the change?

**Comment by Marusyk** [comment]
@suchiagicha Of course, I'll prepare PR. Thanks

**Comment by suchiagicha** [comment]
We have made this fix in 3.2 release . Closing this issue.

**Comment by Marusyk** [comment]
@suchiagicha OK, but can you explain why did you made this changes without my PR? I've tried to help to project that I'm using. But you just threw away my changes and apply yours that is the same.
